addon/components/inputs/datetime.js

```diff
@@ -22,6 +25,15 @@ export default AbstractInput.extend({
 
   // == Computed Properties ====================================================
 
+  @readOnly
+  @computed('cellConfig.renderer.options.defaultToCurrentDateTime')
+  defaultToCurrentDateTime (defaultToCurrentDateTime) {
+    if (isEmpty(defaultToCurrentDateTime)) {
+      return true
+    }
+    return Boolean(defaultToCurrentDateTime)
```

Comment:
Are you allowing strings and numbers?

Reply:
I am but only because I not familiar with the codebase enough to know how to properly restrict the values to only `false` and `true`. I can add PropTypes to the file to indicate that it's supposed to be a boolean, and look into validation to ensure that no non-boolean values are accepted.

Reply:
Personally, I find it's enough to document the types either in code or in the renderers documentation. If you want to go the validation route, you'd have to make an accompanying change in the UISchema2 validation.

Reply:
Well if it's up to me, I'd rather trust developers to use the tool properly. Especially if the tool is documented, and then the developer misuses it, my philosophy is that's on them. I initially looked at the date renderer and didn't see any special type handling in there for the `defaultToCurrentDate` property, so I didn't feel the need to add any here either. I added the `Boolean` cast in a small effort to maintain composure just in case a non-boolean value was given.